### PR TITLE
Openupify

### PR DESCRIPTION
**Pull request by guushoekman**

### CODE REVIEW

File: _layouts/base.html [hunk 1]
```diff
@@ -25,6 +25,7 @@
     <script src="/js/bootstrap-3.3.7.min.js"></script>
     <script src="/js/d3.v4.min.js"></script>
     <script src="/js/main.js"></script>
+    <script src="http://localhost/openup/cross-project-banner/openup-banner.js"></script>
```
Comment (longhotsummer) [inline _layouts/base.html]: ??

**Comment by longhotsummer** [comment]
Looks like an accidental line, but otherwise good.

File: _layouts/base.html (hunk 1 above)
Comment (guushoekman) [reply _layouts/base.html]: Thanks. That was to test the cross project footer with the logo, I accidentally left that in there.

**Comment by longhotsummer** [comment]
🚢

**Pull request merged**